Frameworks/UIKit/WYPopoverController.m

```diff
@@ -1459,10 +1463,15 @@ - (CGRect)innerRect:(CGRect)rect arrowDirection:(WYPopoverArrowDirection)aArrowD
         result.size.height -= borderWidth;
     }
     
+    // The below code is disabled on WINOBJC due to some issue where |viewContentInsets| has a garbage value.
+    // Given that most of the theming/drawing code in WYPopoverController is effectively a no-op in WINOBJC,
+    // just skipping over the below is an adequate (or at least quick) fix.
```

Comment:
Debug builds of the bridge with the WOCCatalog app should show the issue - you'll just see a blank gray popover opening one of the popovers on the "Controls" screen instead of the actual content view controller (without applying this pull request).

I can look at this a little more tomorrow (root cause wise) but it may be related to the KVO of "viewContentInsets": https://github.com/Microsoft/WinObjC/blob/d9578b016df6cfc3822ff409904b69544a2a6ec6/Frameworks/UIKit/WYPopoverController.m#L609